tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -54,6 +81,24 @@ def teardown():
     return ripsaw
 
 
+@pytest.fixture(scope="function")
+def es(request):
```

Comment:
how about calling it `elasticsearch` instead of es`? it will be clearer what this fixture does

Reply:
done